lib/logstash/filters/multiline.rb

```diff
@@ -17,15 +17,15 @@
 # [source,ruby]
 #     filter {
 #       multiline {
-#         type => "type"
 #         pattern => "pattern, a regexp"
 #         negate => boolean
 #         what => "previous" or "next"
 #       }
 #     }
 #
-# The `pattern` should be a regexp which matches what you believe to be an indicator
-# that the field is part of an event consisting of multiple lines of log data.
+# The `pattern` should be a regexp (<<plugins-filters-grok,grok>> patterns are
```

Comment:
Please confirm that this kind of Logstash-internal link will actually work. I see it being used in [geoip.rb](https://github.com/logstash-plugins/logstash-filter-geoip/blob/v2.0.5/lib/logstash/filters/geoip.rb#L14) to link to the elasticsearch output but I can't see that it's documented anywhere.